backend/src/main/java/hu/flowacademy/konyhatunder/model/Recipe.java

```diff
@@ -16,8 +17,6 @@
 @NoArgsConstructor
 public class Recipe {
     @Id
-    @GeneratedValue(generator = "uuid")
```

Comment:
En meghagynam az ID generalast a Hibernate/DB hataskoreben, hacsak nincs valami alapos okotok ra hogy nem igy csinaljatok.

Reply:
Illetve megfontolnam hogy ID-nak Long-ot hasznaljak ha nincs time stamp, mert akkro natrual ordering is lenne az id alapjan, es le lehetne kerni az utolso x receptet.

Reply:
Csak a receptnél vettük ki, azért, mert a recept mentésnél tudnunk kell az ID-t, hogy az AmountIngredient-nek oda tudjuk adni a jelenlegi receptet. Másképp nem tudnánk az id-t.

Reply:
Ha elobb mentenetek le a receptet, es aztan passzolnatok a visszakapott ID-t az AI-nak?

Reply:
Jogos! Át is írtam.